client/src/components/PortfolioFilters.tsx

```diff
@@ -142,8 +169,10 @@ export const PortfolioFilters = React.memo(
           </button>
           <FilterAccordion
             title={i18n._(t`Landlord`)}
-            subtitle={i18n._(t`Officer/Owner`)}
-            infoOnClick={() => setShowOwnerModal(true)}
+            subtitle={i18n._(t`Person/Entity`)}
+            infoLabel={i18n._(t`Who are they?`)}
+            // TODO: add analytics event for onInfoClick
```

Comment:
only request is to move this todo into the outstanding shortcut cards (along w/ other todos)